tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3493,4 +3493,335 @@ function test_get_styles_for_block_with_content_width() {
 		$style_rules = $theme_json->get_styles_for_block( $metadata );
 		$this->assertSame( $expected, $root_rules . $style_rules );
 	}
+
+	/**
+	 * @dataProvider data_generate_spacing_scale_fixtures
+	 *
+	 * @ticket 56467
```

Comment:
`@ticket` should precede `@dataProvider`, and the test description should be the first line.

[See here](https://core.trac.wordpress.org/ticket/56070) for the order of descriptions and annotations.

🔢 Applies to other instances in the PR.

Reply:
done